src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -72,25 +70,33 @@ public void TestInitialize()
         [TestMethod]
         public void Constructor_ThrowsOnNull()
         {
-            Action op = () => new SonarQubeIssuesProvider(null, mockTracker.Object, mockTimerFactory.Object);
+            string validProjectKey = "key1";
+
```

Comment:
It would probably be easier to review the product code changes first, then look at the tests.